ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -727,6 +734,14 @@ nnfw_invoke_internal (const nnfw_pdata * pdata,
   nnfw_internal_stats.total_invoke_latency += stop_time - start_time;
   nnfw_internal_stats.total_invoke_num += 1;
 
+#if (DBG)
+  g_message ("Invoke() is finished: %" G_GINT64_FORMAT ", model path: %s", stop_time - start_time, pdata->model_file);
+  g_message ("%" G_GINT64_FORMAT " invoke average %" G_GINT64_FORMAT ", total overhead %" G_GINT64_FORMAT,
```

Comment:
How about specifying their units? e.g., `10 ms`

Reply:
Sounds good.
Since it also can be applied at `tensor_filter_tensorflow_lite.cc`, a commit has been appended :)